asm/internal/ll.bnf

```diff
@@ -1207,7 +1209,7 @@ GetElementPtrInst
 
 Indices
 	: empty
-	| "," IndexList   << $1, nil >>
+	| "," IndexList CommaAttachedMDs  << $1, nil >>
```

Comment:
Indicies is not an instruction, and should not have attached metadata. The `CommaAttachedMDs` non-terminal belongs to the `GetElementPtrInst` production rule.

Or did you place it here to prevent a LR1 collision? If so annotate with a comment along the lines of:
```go
// TODO: Clean up when the parser generator no longer introduces ambiguities
// through the limitation of 1 rune lookahead.
//
// Structured in this way to allow for naiive 1 rune lookahead parser
// generators.
```